sdk-ext-lib/src/main/java/cash/z/ecc/sdk/fixture/WalletAddressesFixture.kt

```diff
@@ -1,22 +1,22 @@
 package cash.z.ecc.sdk.fixture
 
+import cash.z.ecc.sdk.model.WalletAddress
 import cash.z.ecc.sdk.model.WalletAddresses
 
 object WalletAddressesFixture {
     // These fixture values are derived from the secret defined in PersistableWalletFixture
-
-    const val UNIFIED = "Unified GitHub Issue #161"
-    const val SHIELDED_ORCHARD = "Shielded Orchard GitHub Issue #161"
-    @Suppress("MaxLineLength")
-    const val SHIELDED_SAPLING = "ztestsapling1475xtm56czrzmleqzzlu4cxvjjfsy2p6rv78q07232cpsx5ee52k0mn5jyndq09mampkgvrxnwg"
-    const val TRANSPARENT = "tmXuTnE11JojToagTqxXUn6KvdxDE3iLKbp"
     const val VIEWING_KEY = "03feaa290589a20f795f302ba03847b0a6c9c2b571d75d80bc4ebb02382d0549da"
 
-    fun new(
-        unified: String = UNIFIED,
-        shieldedOrchard: String = SHIELDED_ORCHARD,
-        shieldedSapling: String = SHIELDED_SAPLING,
-        transparent: String = TRANSPARENT,
+    suspend fun new(
+        unified: String = WalletAddressFixture.UNIFIED_ADDRESS_STRING,
+        shieldedOrchard: String = WalletAddressFixture.SHIELDED_ORCHARD_ADDRESS_STRING,
```

Comment:
According to what I know, there's no such "orchard address". Users receive orchard funds through UAs only.

Reply:
I pushed a change removing the orchard address.

We should also coordinate with @geffenz to make sure that references to an orchard address are removed from the UI designs.